### DISCUSSION TITLE
How to enable version 2 (OpenGL) rendering?

### DISCUSSION THREAD

**Discussion by jfburke3** [osmandapp/OsmAnd#16064] [Q&A]
Before I try to open an Issue for this, I want to be sure I'm not missing something obvious.  On my  Samsung S22 phone, and Samsung S7 tablet, running recent "nightly" builds of OsmAnd (Android of course), I cannot enable Version 2 (OpenGL) rendering.  The radio selector is there, but tapping it does nothing.  On my Samsung S20, running the latest Play store beta version,  I can switch back and forth between Version 1 and Version 2 with no problem.  Is there another setting somewhere that I need to toggle to allow Version 2, or is this a bug, or....?

**Comment by anolting** [comment]
Any dev willing to help about this question as I'm facing it also.
